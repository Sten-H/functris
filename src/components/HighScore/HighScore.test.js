import * as React from 'react';
import * as enzyme from 'enzyme';
import { HighScore } from "./index";

describe('HighScore component', () => {
    it('should match snapshot', () => {
        const component = enzyme.shallow(<HighScore />);
        expect(component).toMatchSnapshot()
    })
});