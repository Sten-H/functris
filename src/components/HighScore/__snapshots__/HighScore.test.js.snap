// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighScore component should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h4>
          High Score
</h4>,
        <Table
          bordered={false}
          bsClass="table"
          className="hs-table"
          condensed={true}
          hover={true}
          responsive={false}
          striped={false}
>
          <thead>
                    <tr>
                              <th
                                        className="text-muted"
                              >
                                        #
                              </th>
                              <th
                                        className="text-muted"
                              >
                                        NAME
                              </th>
                              <th
                                        className="text-muted"
                              >
                                        SCORE
                              </th>
                    </tr>
          </thead>
          <tbody>
                    <tr>
                              <td>
                                        1
                              </td>
                              <td>
                                        M4RK
                              </td>
                              <td>
                                        120000
                              </td>
                    </tr>
                    <tr>
                              <td>
                                        2
                              </td>
                              <td>
                                        J4C0B
                              </td>
                              <td>
                                        115000
                              </td>
                    </tr>
                    <tr>
                              <td>
                                        3
                              </td>
                              <td>
                                        GRS
                              </td>
                              <td>
                                        105000
                              </td>
                    </tr>
          </tbody>
</Table>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "High Score",
        },
        "ref": null,
        "rendered": "High Score",
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "bordered": false,
          "bsClass": "table",
          "children": Array [
            <thead>
              <tr>
                            <th
                                          className="text-muted"
                            >
                                          #
                            </th>
                            <th
                                          className="text-muted"
                            >
                                          NAME
                            </th>
                            <th
                                          className="text-muted"
                            >
                                          SCORE
                            </th>
              </tr>
</thead>,
            <tbody>
              <tr>
                            <td>
                                          1
                            </td>
                            <td>
                                          M4RK
                            </td>
                            <td>
                                          120000
                            </td>
              </tr>
              <tr>
                            <td>
                                          2
                            </td>
                            <td>
                                          J4C0B
                            </td>
                            <td>
                                          115000
                            </td>
              </tr>
              <tr>
                            <td>
                                          3
                            </td>
                            <td>
                                          GRS
                            </td>
                            <td>
                                          105000
                            </td>
              </tr>
</tbody>,
          ],
          "className": "hs-table",
          "condensed": true,
          "hover": true,
          "responsive": false,
          "striped": false,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": <tr>
                <th
                                className="text-muted"
                >
                                #
                </th>
                <th
                                className="text-muted"
                >
                                NAME
                </th>
                <th
                                className="text-muted"
                >
                                SCORE
                </th>
</tr>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <th
                    className="text-muted"
>
                    #
</th>,
                  <th
                    className="text-muted"
>
                    NAME
</th>,
                  <th
                    className="text-muted"
>
                    SCORE
</th>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "#",
                    "className": "text-muted",
                  },
                  "ref": null,
                  "rendered": "#",
                  "type": "th",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "NAME",
                    "className": "text-muted",
                  },
                  "ref": null,
                  "rendered": "NAME",
                  "type": "th",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "SCORE",
                    "className": "text-muted",
                  },
                  "ref": null,
                  "rendered": "SCORE",
                  "type": "th",
                },
              ],
              "type": "tr",
            },
            "type": "thead",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <tr>
                  <td>
                                    1
                  </td>
                  <td>
                                    M4RK
                  </td>
                  <td>
                                    120000
                  </td>
</tr>,
                <tr>
                  <td>
                                    2
                  </td>
                  <td>
                                    J4C0B
                  </td>
                  <td>
                                    115000
                  </td>
</tr>,
                <tr>
                  <td>
                                    3
                  </td>
                  <td>
                                    GRS
                  </td>
                  <td>
                                    105000
                  </td>
</tr>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": "0",
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <td>
                      1
</td>,
                    <td>
                      M4RK
</td>,
                    <td>
                      120000
</td>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": 1,
                    },
                    "ref": null,
                    "rendered": 1,
                    "type": "td",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "M4RK",
                    },
                    "ref": null,
                    "rendered": "M4RK",
                    "type": "td",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": 120000,
                    },
                    "ref": null,
                    "rendered": 120000,
                    "type": "td",
                  },
                ],
                "type": "tr",
              },
              Object {
                "instance": null,
                "key": "1",
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <td>
                      2
</td>,
                    <td>
                      J4C0B
</td>,
                    <td>
                      115000
</td>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": 2,
                    },
                    "ref": null,
                    "rendered": 2,
                    "type": "td",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "J4C0B",
                    },
                    "ref": null,
                    "rendered": "J4C0B",
                    "type": "td",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": 115000,
                    },
                    "ref": null,
                    "rendered": 115000,
                    "type": "td",
                  },
                ],
                "type": "tr",
              },
              Object {
                "instance": null,
                "key": "2",
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <td>
                      3
</td>,
                    <td>
                      GRS
</td>,
                    <td>
                      105000
</td>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": 3,
                    },
                    "ref": null,
                    "rendered": 3,
                    "type": "td",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "GRS",
                    },
                    "ref": null,
                    "rendered": "GRS",
                    "type": "td",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": 105000,
                    },
                    "ref": null,
                    "rendered": 105000,
                    "type": "td",
                  },
                ],
                "type": "tr",
              },
            ],
            "type": "tbody",
          },
        ],
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h4>
            High Score
</h4>,
          <Table
            bordered={false}
            bsClass="table"
            className="hs-table"
            condensed={true}
            hover={true}
            responsive={false}
            striped={false}
>
            <thead>
                        <tr>
                                    <th
                                                className="text-muted"
                                    >
                                                #
                                    </th>
                                    <th
                                                className="text-muted"
                                    >
                                                NAME
                                    </th>
                                    <th
                                                className="text-muted"
                                    >
                                                SCORE
                                    </th>
                        </tr>
            </thead>
            <tbody>
                        <tr>
                                    <td>
                                                1
                                    </td>
                                    <td>
                                                M4RK
                                    </td>
                                    <td>
                                                120000
                                    </td>
                        </tr>
                        <tr>
                                    <td>
                                                2
                                    </td>
                                    <td>
                                                J4C0B
                                    </td>
                                    <td>
                                                115000
                                    </td>
                        </tr>
                        <tr>
                                    <td>
                                                3
                                    </td>
                                    <td>
                                                GRS
                                    </td>
                                    <td>
                                                105000
                                    </td>
                        </tr>
            </tbody>
</Table>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "High Score",
          },
          "ref": null,
          "rendered": "High Score",
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "bordered": false,
            "bsClass": "table",
            "children": Array [
              <thead>
                <tr>
                                <th
                                                className="text-muted"
                                >
                                                #
                                </th>
                                <th
                                                className="text-muted"
                                >
                                                NAME
                                </th>
                                <th
                                                className="text-muted"
                                >
                                                SCORE
                                </th>
                </tr>
</thead>,
              <tbody>
                <tr>
                                <td>
                                                1
                                </td>
                                <td>
                                                M4RK
                                </td>
                                <td>
                                                120000
                                </td>
                </tr>
                <tr>
                                <td>
                                                2
                                </td>
                                <td>
                                                J4C0B
                                </td>
                                <td>
                                                115000
                                </td>
                </tr>
                <tr>
                                <td>
                                                3
                                </td>
                                <td>
                                                GRS
                                </td>
                                <td>
                                                105000
                                </td>
                </tr>
</tbody>,
            ],
            "className": "hs-table",
            "condensed": true,
            "hover": true,
            "responsive": false,
            "striped": false,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": <tr>
                  <th
                                    className="text-muted"
                  >
                                    #
                  </th>
                  <th
                                    className="text-muted"
                  >
                                    NAME
                  </th>
                  <th
                                    className="text-muted"
                  >
                                    SCORE
                  </th>
</tr>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <th
                      className="text-muted"
>
                      #
</th>,
                    <th
                      className="text-muted"
>
                      NAME
</th>,
                    <th
                      className="text-muted"
>
                      SCORE
</th>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "#",
                      "className": "text-muted",
                    },
                    "ref": null,
                    "rendered": "#",
                    "type": "th",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "NAME",
                      "className": "text-muted",
                    },
                    "ref": null,
                    "rendered": "NAME",
                    "type": "th",
                  },
                  Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "SCORE",
                      "className": "text-muted",
                    },
                    "ref": null,
                    "rendered": "SCORE",
                    "type": "th",
                  },
                ],
                "type": "tr",
              },
              "type": "thead",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <tr>
                    <td>
                                        1
                    </td>
                    <td>
                                        M4RK
                    </td>
                    <td>
                                        120000
                    </td>
</tr>,
                  <tr>
                    <td>
                                        2
                    </td>
                    <td>
                                        J4C0B
                    </td>
                    <td>
                                        115000
                    </td>
</tr>,
                  <tr>
                    <td>
                                        3
                    </td>
                    <td>
                                        GRS
                    </td>
                    <td>
                                        105000
                    </td>
</tr>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": "0",
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <td>
                        1
</td>,
                      <td>
                        M4RK
</td>,
                      <td>
                        120000
</td>,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": null,
                      "nodeType": "host",
                      "props": Object {
                        "children": 1,
                      },
                      "ref": null,
                      "rendered": 1,
                      "type": "td",
                    },
                    Object {
                      "instance": null,
                      "key": null,
                      "nodeType": "host",
                      "props": Object {
                        "children": "M4RK",
                      },
                      "ref": null,
                      "rendered": "M4RK",
                      "type": "td",
                    },
                    Object {
                      "instance": null,
                      "key": null,
                      "nodeType": "host",
                      "props": Object {
                        "children": 120000,
                      },
                      "ref": null,
                      "rendered": 120000,
                      "type": "td",
                    },
                  ],
                  "type": "tr",
                },
                Object {
                  "instance": null,
                  "key": "1",
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <td>
                        2
</td>,
                      <td>
                        J4C0B
</td>,
                      <td>
                        115000
</td>,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": null,
                      "nodeType": "host",
                      "props": Object {
                        "children": 2,
                      },
                      "ref": null,
                      "rendered": 2,
                      "type": "td",
                    },
                    Object {
                      "instance": null,
                      "key": null,
                      "nodeType": "host",
                      "props": Object {
                        "children": "J4C0B",
                      },
                      "ref": null,
                      "rendered": "J4C0B",
                      "type": "td",
                    },
                    Object {
                      "instance": null,
                      "key": null,
                      "nodeType": "host",
                      "props": Object {
                        "children": 115000,
                      },
                      "ref": null,
                      "rendered": 115000,
                      "type": "td",
                    },
                  ],
                  "type": "tr",
                },
                Object {
                  "instance": null,
                  "key": "2",
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <td>
                        3
</td>,
                      <td>
                        GRS
</td>,
                      <td>
                        105000
</td>,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": null,
                      "nodeType": "host",
                      "props": Object {
                        "children": 3,
                      },
                      "ref": null,
                      "rendered": 3,
                      "type": "td",
                    },
                    Object {
                      "instance": null,
                      "key": null,
                      "nodeType": "host",
                      "props": Object {
                        "children": "GRS",
                      },
                      "ref": null,
                      "rendered": "GRS",
                      "type": "td",
                    },
                    Object {
                      "instance": null,
                      "key": null,
                      "nodeType": "host",
                      "props": Object {
                        "children": 105000,
                      },
                      "ref": null,
                      "rendered": 105000,
                      "type": "td",
                    },
                  ],
                  "type": "tr",
                },
              ],
              "type": "tbody",
            },
          ],
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
